tensorflow_datasets/image_classification/bigearthnet.py

```diff
@@ -114,7 +114,12 @@ def __init__(self, selection=None, **kwargs):
       raise ValueError('selection must be one of %s' % _DATA_OPTIONS)
 
     v100 = tfds.core.Version(
-        '1.0.0', 'New split API (https://tensorflow.org/datasets/splits)')
+        '1.0.0')
```

Comment:
The review from a PR should apply to all PR. In this case the formatting isn't correct.

Reply:
@Conchylicultor Apologies for that! Should I open another PR for fixing the formatting as this has been merged?